src/pages/MyOffersPage.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Paper } from "@material-ui/core";
+import CenteredComponent from "../components/HomePage/CenteredComponent";
+import { MainMask } from "../components/HomePage/MainMask";
+
+export const MyOffersPage = () => (
+    <>
+        <MainMask />
+        <CenteredComponent>
+            <Paper style={{ width: "60%", padding: "24px 72px", boxSizing: "content-box" }}>
+                <h1>My Offers Page</h1>
```

Comment:
This title will be in conflict with the PageLayout title in mobile. I'd say you wait for the work I'm doing in the Reactive Pattern (I'm refactoring all pages that have desktop/mobile versions) and deal with this logic after that.